src/Engine/UnitMods/LuckBaseModifier.java

```diff
@@ -0,0 +1,20 @@
+package Engine.UnitMods;
+
+import Engine.Combat.StrikeParams;
+
+public class LuckBaseModifier implements UnitModifierWithDefaults
```

Comment:
Could use a docstring to explain intent. If I'm understanding correctly, it coerces the luck value in all combat to the provided luck value. If so, calling it a "Base" modifier seems misleading, as that might just imply changing the luck floor.

Reply:
Nope. The "base" luck is the "zero point" that the good luck and bad luck vary from.

e.g. AW1 Sonja has single-roll -15 to +9, so she has 25 good luck and -15 base luck.

Reply:
Yup. AW1 is the only game that has a single RN for bad and good luck.

> Still think a docstring would be helpful.

I am mildly resistant to the idea because it's a class that **just** modifies a variable (which is in the name of the class), and that variable is documented... so it ends up being kind of duplicate info.

> do we need to verify which unit is attacking in `modifyUnitAttack` to make sure we don't also buff enemies

Nope. That's only for the context hook, since that lets you swap the combatants around. Otherwise, you know which combatant is "yours" based on the hook being called.